azerbaijani/nağılşmd

```diff
@@ -0,0 +1,16 @@
+Biri var idi, biri yox idi. Bir ayı, günü keçmiş kişi var idi. Bir arvadı var idi.
```

Comment:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.